lib/services/local/inventory.go

```diff
@@ -0,0 +1,149 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package local
+
+import (
+	"context"
+
+	"github.com/gravitational/trace"
+
+	apidefaults "github.com/gravitational/teleport/api/defaults"
+	"github.com/gravitational/teleport/api/internalutils/stream"
```

Comment:
Something still feels off about this. If it's truly internal, then modules other than API shouldn't know anything about it. Is the intent here to signal that this is not safe for consumers of the API module to use (other than Teleport itself)?

Reply:
> Is the intent here to signal that this is not safe for consumers of the API module to use (other than Teleport itself)?

Yeah.  The intent is to communicate the fact that it might not adhere to the stability guarantees of the `api` package.  We bikeshedded on the naming during the PR that introduced that package.  Not sure there is a "right" name.  The go compiler prevents us from using `internal`, and folks found `unstable` too scary sounding.  I ended up deciding to not let the perfect be the enemy of the good, and just use something that invoked the `internal` naming convention of go while still being importable by `lib`.  Since it _is_ unstable, we can always change it later if someone thinks of a better name.